File: app/app/Model/Auth/Response/MemberGetCommentResponse.swift
struct MemberGetCommentResponse: Codable, Identifiable {

    let memberId: Int64
    let thumbnail: String?
    let nickname: String
    let comment: String
    let gender: String
    let age: Int
    let likes: Int64
    let distance: Double?
    let createdAt: String

    var id: Int64 { memberId }
}

File: server/src/main/kotlin/com/openmpy/server/member/dto/response/MemberGetCommentResponse.kt
package com.openmpy.server.member.dto.response

import com.openmpy.server.member.domain.type.Gender
import java.time.LocalDateTime

data class MemberGetCommentResponse(

    val memberId: Long,
    val thumbnail: String?,
    val nickname: String,
    val comment: String,
    val gender: Gender,
    val age: Int,
    val likes: Long,
    val distance: Double?,
    val updatedAt: LocalDateTime,
)